src/components/survey/__tests__/sorting-hat-reducer.test.js

```diff
@@ -0,0 +1,219 @@
+  test('should close sorting hat for load action with undefined subscriber after finished loading', () => {
+    expect(
+      sortingHatReducer(sortingHatInitialState, {
+        type: 'load',
+        loadingSubscriber: 'false',
+      }),
+    ).toStrictEqual({
+      ...sortingHatInitialState,
+      question: defaultFirstQuestion,
+      closed: true,
+    })
+  })
+
+  test('should add question property to state on load action while loading subscriber', () => {
+    expect(
+      sortingHatReducer(sortingHatInitialState, {
+        type: 'load',
+        loadingSubscriber: 'true',
+      }),
+    ).toStrictEqual({...sortingHatInitialState, question: defaultFirstQuestion})
+  })
+
+  test('should close survey for subscriber with survey complete', () => {
+    const subscriber = {
+      id: 'testCompleteSurvey',
+      email: 'test@egghead.io',
+      attributes: {sorting_hat_finished_at: 'today'},
+    }
+    expect(
+      sortingHatReducer(sortingHatInitialState, {
+        type: 'load',
+        subscriber,
+        loadingSubscriber: 'false',
+      }),
+    ).toStrictEqual({
+      ...sortingHatInitialState,
+      question: defaultFirstQuestion,
+      closed: true,
+    })
+  })
+
+  test('should initialize survey state to default first question for subscriber with incomplete survey', () => {
+    const subscriber = {
+      id: 'testIncompleteSurvey',
+      email: 'test@egghead.io',
+    }
+    const defaultFirstQuestionKey = sortingHatInitialState.currentQuestionKey
+    expect(
+      sortingHatReducer(sortingHatInitialState, {
+        type: 'load',
+        subscriber,
+        loadingSubscriber: 'false',
+      }),
+    ).toStrictEqual({
+      ...sortingHatInitialState,
+      closed: false,
+      question: defaultFirstQuestion,
+      subscriber,
+      currentQuestionKey: defaultFirstQuestionKey,
+    })
+  })
+})
+
+describe('answered action', () => {
+  const subscriber = {
+    id: 'testIncompleteSurvey',
+    email: 'test@egghead.io',
+  }
+  const loadedState = sortingHatReducer(sortingHatInitialState, {
+    type: 'load',
+    subscriber,
+    loadingSubscriber: 'false',
+  })
+  test('initial state is loaded and answering first question updates state appropriately', () => {
+    const nextQuestionKey = 'level_up_reason'
+    const nextQuestion = sortingHatInitialState.data[nextQuestionKey]
+    expect(
+      sortingHatReducer(loadedState, {
+        type: 'answered',
+        answer: 'leveling_up',
+      }),
+    ).toStrictEqual({
+      ...loadedState,
+      question: nextQuestion,
+      currentQuestionKey: nextQuestionKey,
+      answers: {biggest_path: 'leveling_up'},
+      subscriber: {
+        ...subscriber,
+        sorting_hat_started_at: Math.round(Date.now() / 1000),
+      },
+    })
+    expect(track).toHaveBeenCalledWith(
+      'answered survey question',
+      expect.anything(),
+    )
+    expect(track).toHaveBeenLastCalledWith('started survey', expect.anything())
```

Comment:
3 expects is 3 tests and should be treated as such, even if they have the same setup. You can use `beforeEach` in a `describe` for the block to reduce repetition.